src/renderer/components/icon/icon.tsx

```diff
@@ -87,7 +87,7 @@ export class Icon extends React.PureComponent<IconProps> {
 
     // render as inline svg-icon
     if (svg) {
-      const svgIconText = require(`!!raw-loader!./${svg}.svg`).default;
+      const svgIconText = require("!!raw-loader!./" + svg + ".svg").default;
```

Comment:
Is there any difference? :)

Reply:
No, but I think I had already reverted this by the time... lol.